packaging/specifiers.py

```diff
@@ -97,7 +106,7 @@ class _IndividualSpecifier(BaseSpecifier):
 
     def __init__(self, spec="", prereleases=None):
         # type: (str, Optional[bool]) -> None
-        match = self._regex.search(spec)
+        match = self._regex.search(spec)  # type: ignore
```

Comment:
Why is this necessary?

Reply:
Good point, it isn't. I've removed the comment and am now defining `_regex` on the class with a type.